packages/calcite-components/src/assets/styles/includes.scss

```diff
@@ -97,10 +97,8 @@
 }
 
 @mixin base-component() {
-  :host([hidden]) {
```

Comment:
Since the internal `hide-item` attribute is only applicable to certain components, can you create a separate mixin and include in applicable children?